frontend/src/utils.js

```diff
@@ -0,0 +1,21 @@
+const capitalize = (word) => {
```

Comment:
Check lodash : https://lodash.com/docs/4.17.15#upperFirst or https://lodash.com/docs/4.17.15#capitalize

Reply:
I mean I think that we can use directly the lodash functions instead of creating our own (and test them)

Reply:
Good suggestion, done. https://github.com/dr4fters/dr4ft/pull/990/commits/8b109b7d4174f1b5aa548ce38f76940f75e97341

Note that I left `toTitleCase()` because of this https://github.com/dr4fters/dr4ft/blob/82f46024e8680d4dac3df7e906a64e3a66efaebd/frontend/src/lobby/Set.jsx#L22

There's [`startCase()`](https://lodash.com/docs/4.17.15#startCase) in lodash which is pretty close to what `toTitleCase()` does but not quite.